content/javascript/concepts/window/terms/clearTimeout/clearTimeout.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: 'clearTimeout()'
+Description: 'Clears a previously scheduled timeout function.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'Arguments'
+  - 'Functions'
+  - 'Parameters'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**`clearTimeout()`** is a window method that allows to clear a previously scheduled `setTimeout()` from running.
```

Comment:
```suggestion
**`clearTimeout()`** is a window method that stops a previously scheduled `[setTimeout()](https://www.codecademy.com/resources/docs/javascript/window/setTimeout)` from running.
```

Reply:
I had to remove the backticks otherwise the link would not work correctly